mmd_tools/operators/misc.py

```diff
@@ -50,27 +50,35 @@ def execute(self, context):
         obj = context.active_object
         root = mmd_model.Model.findRoot(obj)
         if root is None:
-            self.report({ 'ERROR' }, 'Select a MMD model') 
-            return { 'CANCELLED' } 
+            # self.report({ 'ERROR' }, 'Select a MMD model') 
+            # return { 'CANCELLED' }
+            # find all meshes in the active layer
+            act_layer = bpy.context.scene.active_layer
+            meshes_list = [o for o in bpy.data.objects
+                           if o.layers[act_layer] and o.type == 'MESH' and o.mmd_type == 'NONE']
+            active_mesh = meshes_list[0]
```

Comment:
FWIW I was able to connect a debugger and step through the mmd_tools code with blender running: https://github.com/ptrthomas/blender_mmd_tools/blob/dev-temp/debug_mode_plugin/__init__.py

If I get around to figuring out unit tests, I'll let you folks know.